src/styles/style.scss

```diff
@@ -0,0 +1,98 @@
+  @import '/src/styles/stars';
+  @import '/src/general';
+
+  .card {
+    padding: 32px 16px 16px;
+    width: 200px;
+    border-radius: 5px;
+    border: 1px solid #f3f3f3;
+    background: #fff;
+
+    &__image {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      margin: 0 auto 40px;
+      width: 160px;
+      height: 134px;
+    }
+
+    &__header {
+      margin-bottom: 4px;
+      color: var(--main-accent, #060B35);
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+    }
+
+    &__product-code {
+      margin-bottom: 16px;
+      color: var(--secondary, #616070);
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+    }
+
+    &__reviews-container {
+      display: flex;
+      justify-content: space-between;
+      align-items: flex-end;
+    }
+
+    &__reviews {
+      display: flex;
+      flex-direction: column;
+      justify-content: flex;
+      color: var(--main-accent, #060B35);
+      text-align: right;
+      font-size: 10px;
+    }
+
+    &__price-container {
+      margin-top: 24px;
```

Comment:
Pleae use either margin bottom of margin top to make indentations between elements. do not mix them, it could cause margin collapse